api/v1alpha1/types.go

```diff
@@ -30,6 +30,37 @@ const (
 	MeshGatewayModeRemote MeshGatewayMode = "remote"
 )
 
+// ExposeConfig describes HTTP paths to expose through Envoy outside of Connect.
```

Comment:
nit: looks like the MeshGateway stuff above here isn't used at all and can be deleted